### DISCUSSION TITLE
How to give multiple action to single shape in Visio

### DISCUSSION THREAD

**Discussion by JAYASURIYARS1799** [MicrosoftDocs/office-developer-client-docs#677] [Q&A] [answered]
In Visio software I make a master shape of “minus” symbol,now I want to add 3 action for that minus shape . I want to give ” Plus”,”Multiple”&”Divided” , .. I make individual shape of that three symbol. How I give this 3 action in that “minus” master shape ..
I need some video reference file also , I think you can only help out in this topic.. I’m waiting for you’re response
@lindalu-MSFT

**Comment by lindalu-MSFT** [comment, marked as answer]
@JAYASURIYARS1799 For Visio Q&A, try either Stack Overflow https://stackoverflow.com/search?q=visio or Microsoft Q&A https://learn.microsoft.com/en-us/answers/tags/150/office-visio. Also, please remove your PII (Personally Identifiable Information) as in your private email from your question.
